docs/administration-guide/10-single-sign-on.md

```diff
@@ -20,34 +18,47 @@ Now existing Metabase users signed into a Google account that matches their Meta
 
 ###  Enabling account creation with Google Sign-In
 
-If you’ve added your Google client ID to your Metabase settings you can also let users sign up on their own without creating accounts for them.
+If you’ve added your Google client ID to your Metabase settings, you can also let users sign up on their own without creating accounts for them.
 
 To enable this, go to the Google Sign-In configuration page, and specify the email domain you want to allow. For example, if you work at WidgetCo you could enter `widgetco.com` in the field to let anyone with a company email sign up on their own.
 
-Note: Metabase accounts created with Google Sign-In do not have passwords and must use Google to sign in to Metabase.
+Note that Metabase accounts created with Google Sign-In do not have passwords and must use Google to sign in to Metabase.
 
+## Enabling LDAP authentication
 
-### Enabling LDAP authentication
+If your organization uses LDAP, enabling it in Metabase will allow your users to log in via their LDAP credentials.
 
-If your organization uses LDAP, and you want to allow your users to log in via their LDAP credentials, you can do so as follows.
+To get started, click the **Configure** button in the LDAP section of the Admin Panel's Authentication page, where you'll fill out the form with the information about your LDAP server — hostname, port, security settings, and the LDAP admin username and password — and save your changes.
 
-Click the `Configure` button in the LDAP section of the Authentication page, and you'll see this form:
+Metabase will pull out three main attributes from your LDAP directory - email (defaulting to the `mail` attribute), first name (defaulting to the `givenName` attribute), and last name (defaulting to the `sn` attribute). If your LDAP setup uses other attributes for these, you can edit this under the "Attributes" portion of the form.
 
-![Authentication](./images/ldap-form.png)
+![Attributes](./images/ldap-attributes.png)
 
-Click the toggle at the top of the form to enable LDAP, then fill in the form with the information about your LDAP server.
+Your LDAP directory must have the email field populated for each entry that will become a Metabase user, otherwise Metabase won't be able to create the account, nor will that person be able to log in. If either name field is missing, Metabase will use a default of "Unknown," and the name can be changed manually in the person's account settings.
```

Comment:
```suggestion
Your LDAP directory must have the email field populated for each entry that will become a Metabase user, otherwise Metabase won't be able to create the account, nor will that person be able to log in. If either name field is missing, Metabase will use a default of "Unknown," and the person can change their name in their [account settings](../users-guide/account-settings.md).
```
Always be on the lookout for sprinkling in some helpful links.